lib/travis/addons/handlers/metrics.rb

```diff
@@ -1,11 +1,15 @@
-require 'travis/metrics'
-require 'travis/event/handler'
+require 'travis/addons/handlers/base'
+require 'travis/addons/handlers/task'
+require 'travis/addons/model/broadcast'
+require 'travis/rollout'
 
 module Travis
-  module Hub
-    module Event
-      class Metrics < Travis::Event::Handler
-        register :metrics, self
+  module Addons
+    module Handlers
+      class Metrics < Base
+        include Handlers::Task
+
+        Event::Handler.register('metrics', self)
```

Comment:
This should not be needed in this case.

The reason why logsearch does it is because by default the name is snake cased, i.e. `LogSearch` => `log_search`, and I wanted it to show up as `logsearch` (iirc it was in the logs? or maybe the key in `lib/travis/hub/context.rb`? or maybe the config?).

In any case, here the class name matches the event handler name, so it should be fine to drop this line.